ledgerwallet/manifest_toml.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+from typing import Dict, Optional
+
+from ledgerwallet.manifest import AppManifest, icon_from_file
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import toml as tomllib
+
+from ledgerwallet import params
+
+
+class AppManifestToml(AppManifest):
+    def __init__(self, filename):
+        self.path = os.path.dirname(filename)
+        if sys.version_info >= (3, 11):
+            with open(filename, "rb") as f:
+                self.dic = tomllib.load(f)
+        else:
+            with open(filename, "r") as f:
+                self.dic = tomllib.load(f)
+
+    @property
+    def app_name(self) -> str:
```

Comment:
This feels redundant with `AppManifestJson.app_name`. If the behavior does not change across `AppManifest` children, this could be integrated into the (now partial) interface motherclass.